wiki/development/coding-guidelines.md

```diff
@@ -298,7 +302,7 @@ Comments within the code should be used when they are describing a complex and c
 
 Good:
 
-```js
+```sqf
```

Comment:
I assumed it would because it was used somewhere else in that file. Will change.